packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -21,6 +29,94 @@ List<Widget> dialogSubHeader(ThemeData theme, String titleText) {
   ];
 }
 
+/// A standardized dialog with help text and buttons `Reset to default`,
+/// `APPLY` and `CANCEL`.
+class DialogWithDefaults extends StatelessWidget {
```

Comment:
rename to FilterDialogWithDefaults since this combination of actions (apply + cancel + reset to default) only apply to filter dialogs, not all dialogs

Reply:
Right now yes, but I can imagine some settings dialog that would looks this way too.

Reply:
Good point. Maybe something like 'StatefulDialogWithDefaults'?

Reply:
Us discussed offline renamed to StateUpdateDialog